dfastmi/gui/dialog_view_model.py

```diff
@@ -53,8 +54,10 @@ class DialogViewModel(QObject):
     save_plot_changed = pyqtSignal(bool)
     figure_dir_changed = pyqtSignal(str)
     output_dir_changed = pyqtSignal(str)
-    _reference_files: Dict[float, str] = {}
-    _measure_files: Dict[float, str] = {}
+    reference_files_changed = pyqtSignal(str, float, Path)
+    measure_files_changed = pyqtSignal(str, float, Path)
+    _reference_files: Dict[float, Path] = {}
+    _measure_files: Dict[float, Path] = {}
```

Comment:
Link to #98 ... This should be FilenameDict, correct?

Reply:
Ah, we should then change the definition of FilenameDict since that refers to str instead of Path.